src/features/applications/mappers/index.ts

```diff
@@ -89,3 +98,44 @@ const getValue = (bytes: string) => {
     }
   }
 }
+
+export const asMethodDefinitions = (appSpec: Arc32AppSpec): MethodDefinition[] => {
+  return appSpec.contract.methods.map((method) => {
+    const abiMethod = new algosdk.ABIMethod({
+      name: method.name,
+      desc: method.desc,
+      args: method.args,
+      returns: method.returns,
+    })
+    const signature = abiMethod.getSignature()
+    const hint = appSpec.hints ? appSpec.hints[signature] : undefined
+
+    return {
+      name: abiMethod.name,
+      signature: signature,
+      description: abiMethod.description,
+      arguments: abiMethod.args.map((arg, index) => ({
+        index: index + 1,
```

Comment:
```suggestion
        index: index + 1,
```
Non zero based index?